src/components/Menu/__snapshots__/menu.test.js.snap

```diff
@@ -47,7 +47,7 @@ exports[`Popover component matches snapshot (JSX items) 1`] = `
             "element": <div
               data-reach-menu-item=""
               data-valuetext="First"
-              id="option-0--menu--5"
+              id="option-0--menu--7"
```

Comment:
I don't know why this is changing, but it doesn't matter because I'm nuking this snapshot the first opportunity I get around to it.